.github/workflows/test.yaml

```diff
@@ -0,0 +1,32 @@
+
+name: Unit Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
```

Comment:
why so many empty lines in the spec? Is this a norm for github config?

Reply:
Came with the auto-generated template for Go. I removed a few, but I think they're still useful to have in-between steps.